### PR TITLE
Module1

### PR DESCRIPTION
**Pull request by IzyNP**

### CODE REVIEW

File: src/com/izmaylov/module1/service/UserInputService.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+                    break;
+                case 2:
+                    actions.ascendingSort(array);
+                    isSorted = true;
+                    break;
+                default:
+                    System.out.println("Выберите из списка.");
+            }
+        }while (!isSorted);
+        actions.everyThirdNumberInCube(array);
```
Comment (Fabolo12) [inline src/com/izmaylov/module1/service/UserInputService.java]: It shouldn't be in the sort method

**Review by Fabolo12 — APPROVED** [review]
10

File: src/com/izmaylov/module1/service/UserInputService.java (hunk 1 above)
Comment (IzyNP) [reply src/com/izmaylov/module1/service/UserInputService.java]: should i import the class Actions into main and call method there?

Comment (Fabolo12) [reply src/com/izmaylov/module1/service/UserInputService.java]: Yes

**Pull request merged**